base/strings/stringprintf.cc

```diff
@@ -29,7 +29,7 @@ inline int vsnprintfT(char* buffer,
                       size_t buf_size,
                       const char* format,
                       va_list argptr) {
-  return base::vsnprintf(buffer, buf_size, format, argptr);
+  return ::vsnprintf(buffer, buf_size, format, argptr);
```

Comment:
We shouldn't be changing things like this, as this is in upstream as well. Calling base::vsnprintf is preferable

Reply:
This change is needed.

Reply:
Could you please explain why this is needed instead of just saying it is? It seems to me that base::vsnprintf is there for a reason given the comment in `base/strings/string_util.h`

Reply:
The declaration in base/strings/string_util.h has been removed in this CL.

Reply:
Yes. It is updated.